src/js/index.js

```diff
@@ -8,7 +8,8 @@ const sort = {
 	ALPHABETICAL_ASCENDING: (a, b) => a.dataset.name > b.dataset.name ? 1 : -1,
 	ALPHABETICAL_DESCENDING: (a, b) => a.dataset.name < b.dataset.name ? 1 : -1,
 	DIFFICULTY_ASCENDING: (a, b) => a.dataset.difficulty > b.dataset.difficulty ? 1 : -1,
-	DIFFICULTY_DESCENDING: (a, b) => a.dataset.difficulty < b.dataset.difficulty ? 1 : -1
+	DIFFICULTY_DESCENDING: (a, b) => a.dataset.difficulty < b.dataset.difficulty ? 1 : -1,
+	LAST_UPDATED_ON: (a, b) => a.dataset.dateAdded > b.dataset.dateAdded ? 1 : -1
```

Comment:
To keep things matching with our fields name, I propose to use `DATE_ADDED` prefix with the `ASCENDING` / `DESCENDING` suffixes instead.

```suggestion
	DATE_ADDED_ASCENDING: (a, b) => a.dataset.dateAdded > b.dataset.dateAdded ? 1 : -1
	DATE_ADDED_DESCENDING: (a, b) => a.dataset.dateAdded < b.dataset.dateAdded ? 1 : -1
```